src/components/PopupWithConfirmation.js

```diff
@@ -1,10 +1,12 @@
 import Popup from "./Popup.js";
```

Comment:
Needs correction in Popup.js line 17: there is no need to prevent the default event behaviour, since it can be used by other code

Reply:
I removed line 17, "prevent default," from Popup.js.